daras_ai_v2/img_model_settings_widgets.py

```
@@ -513,3 +611,25 @@ def negative_prompt_setting(selected_model: str | None = None):
     )
+
+
+def edit_instruction_setting(selected_models: list[str] | None = None):
+    if selected_models and not any(
+        selected_model in [Img2ImgModels.instruct_pix2pix.name]
+        for selected_model in selected_models
+    ):
+        return
+    gui.caption(
+        """
+            You can also enable ‘Edit Instructions’ to use InstructPix2Pix that allows you to change your generated image output with a follow-up written instruction.
```

Ruff (0.14.5)

String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)